python/paddle/fluid/tests/unittests/test_imperative_auto_mixed_precision.py

```diff
@@ -165,6 +165,34 @@ def run_simple_conv(inp_np, use_scaler=True):
                 np.allclose(outs_with_scaler[1][i][0].numpy(),
                             outs_no_scaler[1][i][0].numpy()), True)
 
+    def test_lamb_grad_clip(self):
+        inp_np = np.random.random(size=[1, 3, 128, 128]).astype(np.float32)
+
+        def run_simple_conv(inp_np, use_scaler=True):
+            paddle.seed(10)
+            paddle.framework.random._manual_program_seed(10)
+            with fluid.dygraph.guard():
```

Comment:
On paddle2.0, should we use `paddle.disable_static(), paddle.enable_static()` instead of `dygraph.guard` ?

Reply:
Thanks, this unittest had be deleted.